tools/cldr-apps/src/org/unicode/cldr/web/STFactory.java

```diff
@@ -1293,84 +1304,6 @@ private void internalSetVoteForValue(User user, String distinguishingXpath, Stri
             stamp.next();
         }
 
-        @Override
```

Comment:
Is this unused? Please explain in a comment on the PR.

Reply:
Yes, it's unused, unreachable code. Though I don't know the history behind WHAT_DELETE and deleteValue, I suspect their original purpose is now accomplished by Abstain.